recipes/tensorly/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "tensorly" %}
+{% set version = "0.4.2" %}
+{% set fn = "{}-{}-py3-none-any.whl".format(name, version) %}
```

Comment:
I don't think I've seen a package that only has a wheel release but no source release. I guess this is ok, but pinging @conda-forge/staged-recipes just in case someone else knows of concerns.

Reply:
Thanks for the review @synapticarbors! 

I followed a previous example where @dougalsutherland built a [recipe from a PyPI wheel](https://github.com/conda-forge/asciimatics-feedstock/pull/2).

Reply:
Thoughts @conda-forge/core?

Reply:
So personally would just build from the source on GitHub. Though did read the linked PR above and guess if it is pure Python, it would work to build from a wheel. That said, agree it would be good to get other's thoughts on this. Thanks for raising @synapticarbors. :)

Reply:
I'm :+1:  for building from source.

Reply:
Something that might be worth noting is that with the PR that I referenced earlier, in the [initial build of the asciimatics recipe](https://github.com/conda-forge/staged-recipes/pull/5697), we tried to build the package from tarball in the GitHub releases, but the following error was generated:

```bash
LookupError: setuptools-scm was unable to detect version for '/private/tmp/pip-02Ke2e-build'
```

I'm not sure if this error is unique to the way that package was originally built, but figured it might be worth mentioning here in case it's something that could happen with other packages.

Reply:
I think that is mainly an issue with `setuptools-scm`.